packages/strapi-plugin-content-manager/admin/src/components/DynamicZone/CategoryItem.js

```diff
@@ -0,0 +1,69 @@
+import React, { useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+import { Collapse } from 'reactstrap';
+import Banner from './Banner';
+import DynamicComponentCard from '../DynamicComponentCard';
+
+const CategoryItem = ({
+  category,
+  components,
+  isOpen,
+  isFirst,
+  onClickToggle,
+  onClickComponent,
+}) => {
+  const [showComponents, setShowComponents] = useState(false);
+  useEffect(() => {
+    if (isOpen) {
+      setShowComponents(true);
+    }
+  }, [isOpen]);
+
+  return (
+    <>
+      <Banner onClickToggle={onClickToggle} isFirst={isFirst} isOpen={isOpen} category={category} />
+      <Collapse
+        isOpen={isOpen}
+        style={{ backgroundColor: '#F3F3F3' }}
```

Comment:
This custom background seems to be useless. Can you remove it please.

Reply:
@HichamELBSI done in 773c0a8a7f3cc738c303f58625cbf6b852104c6d